src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -1844,10 +1846,11 @@ int xccdf_policy_generate_fix(struct xccdf_policy *policy, struct xccdf_result *
 			return 1;
 		}
 
-		if (_write_script_header_to_fd(policy, result, sys, input_file_name, tailoring_file_name, output_fd) != 0) {
-			oscap_list_free(rules_to_fix, NULL);
-			return 1;
-		}
+		if (raw == 0)
+			if (_write_script_header_to_fd(policy, result, sys, input_file_name, tailoring_file_name, output_fd) != 0) {
```

Comment:
When generating Ansible Playbooks the output is invalid because this part contains essential part of the output.

Reply:
Fixed.